classes/pickup/pickup.gd

```diff
@@ -65,6 +68,16 @@ func _pickup_id_setup() -> void:
 			queue_free()
 
 
+func _pickup_sound():
+	if sfx != null:
+		# Find an object we know will survive this object's destruction.
+		var safe_sfx_root = get_parent()
```

Comment:
I'm not so sure this is definitely "safe". I trust you, but I'm wondering why you chose to do this instead of parenting it to the scene root.

Reply:
Hmm. You know what, that's actually a better idea.

Reply:
I can revert the merge if needed.

Reply:
Nevermind, I can't easily do that. Best to just make a new PR.

Reply:
Might be a good idea

Reply:
ah yep, dang :/